Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -66,6 +67,22 @@ def livecheck_formulae(formulae_to_check, args)
       @livecheck_strategy_names.freeze
 
       has_a_newer_upstream_version = false
+
+      if args.json? && !args.quiet? && $stderr.tty?
+        total_formulae = if formulae_to_check == Formula
+          formulae_to_check.count
+        else
+          formulae_to_check.length
+        end
+        $stderr.puts Formatter.headline("Running checks", color: :blue)
+        progress = ProgressBar.create(
+          total:          total_formulae,
+          progress_mark:  "#",
+          remainder_mark: ".",
+          format:         " %t: [%B] %c/%C ",
+          output:         $stderr,
+        )
+      end
```

Comment:
```suggestion
      end

```

I think I suggested this in a previous review but maybe we lost the change along the way. This blank line simply separates this `if` block from the code after it, for the sake of readability.